versioned_docs/version-0.x.0/kubeslice-cli/kubeslice-cli-demo.mdx

```diff
@@ -0,0 +1,348 @@
+# KubeSlice CLI Demo
```

Comment:
Change this title as `Demo using kubeslice-cli`

Reply:
done